spec.bs

```diff
@@ -763,70 +763,258 @@ enrollment and authentication. This extends the
+ensure that it aligns with the transaction details that the merchant initiated
+the transaction for.
 
-     * If the relying party uses the same credentials for a given user across
-         multiple payment instruments, this could expose that linkage to the
-         merchant (as they could track what sets of credentials they see for
-         each instrument they ask about on the backend). See https://github.com/w3c/secure-payment-confirmation/issues/77
+# Privacy Considerations # {#sctn-privacy-considerations}
+
+As this specification builds on top of WebAuthn, the
+[[webauthn-3#sctn-privacy-considerations|WebAuthn Privacy Considerations]]
+are applicable to this specification. The below subsections comprise the
+current Secure Payment Confirmation-specific privacy considerations, where
+this specification diverges from WebAuthn.
+
+## Enrollment in a Cross-Origin iframe ## {#sctn-security-cross-origin-enrollment}
+
+Unlike WebAuthn, this specification allows the creation of credentials in a
+cross-origin iframe (as long as the appropriate
+[[#sctn-permissions-policy|Permission Policy]] is set on the iframe). That is,
+if site A embeds an iframe from site B, with the "[=payment permission
+string|payment=]" policy set, then site B may initiate a credential creation
+for site B within that iframe.
+
+NOTE: Allowing credential creation in cross-origin iframes is currently [under
+      discussion](https://github.com/w3c/webauthn/issues/1656) in the WebAuthn
+      Working Group, and thus may move from this specification to WebAuthn in
+      the future.
+
+A [previously described
+attack](https://github.com/w3c/webauthn/issues/1336#issue-513568855) on this
+feature exists. In it, a malicious iframe (`https://website.tracker`) initiates
+credential creation, attempting to trick the user into thinking that the
+WebAuthn credential creation is for logging into the legitimate parent site
+(`https://example.org`). The user creates the credential. The malicious actor
+can then later utilize `get()` in a cross-origin iframe (again, timing the call
+to coincide with a login attempt to `https://example.org`) to have the user
+accidentally identify themselves to the tracker iframe.
+
+This attack, however, presumes that `https://website.tracker` already has
+script access to `https://example.org` (in order to create the attacker iframe,
+with the appropriate permissions set). If this is true, then
+`https://website.tracker` can already directly track the user, for example by:
+
+* Identifying them from the username they provide when logging in to
+    `https://example.org`, OR
+* Creating a WebAuthn credential for `https://example.org` (in the main frame),
+    saving that to a `https://website.tracker` server, and later using that
+    credential (again in the main frame) to identify and track the user.
 
-* Credential ID(s) as a tracking vector
+<div class="note">
+**TODO**: The described attack does allow `https://website.tracker` to track the
```

Comment:
Nit: I did not follow which attack is being implied here. The attack description may help.